apps/help-center/config.js

```diff
@@ -1,6 +1,9 @@
+/* global helpCenterData */
 window.configData = {
-	env_id: 'production',
-	zendesk_support_chat_key: 'cec07bc9-4da6-4dd2-afa7-c7e01ae73584',
+	env_id: helpCenterData?.isProxied ? 'staging' : 'production',
+	zendesk_support_chat_key: helpCenterData?.isProxied
```

Comment:
For a better readability, I would add these keys to a `CONST` variable describing where they belong to